framework/doc/content/source/linearfvbcs/LinearFVScalarSymmetryBC.md

```diff
@@ -0,0 +1,37 @@
+# LinearFVScalarSymmetryBC
```

Comment:
so should it be a LinearFVAdvectionDiffusionSymmetryBC then?
seems to be hardcoded for diffusion and advection